optimum/exporters/onnx/utils.py

```diff
@@ -131,7 +128,7 @@ def get_decoder_models_for_export(
         `Dict[str, Tuple[Union[`PreTrainedModel`, `TFPreTrainedModel`], `OnnxConfig`]: A Dict containing the model and
         onnx configs for the encoder and decoder parts of the model.
     """
-    models_for_export = dict()
+    models_for_export = {}
 
     models_for_export["decoder_model"] = (
         model,
```

Comment:
just below I used `config.__class__`, but can we replace it by `config.with_behavior()` now?

Reply:
Yes, fixed it!

Reply:
great